app/controllers/chairs_controller.rb

```diff
@@ -0,0 +1,147 @@
+class ChairsController < ApplicationController
+  before_action :set_chair, only: [:show, :accept_request, :remove_from_chair, :destroy]
+
+  before_action :authorize_admin, only: [:show, :accept_request, :remove_from_chair]
+  before_action :authorize_superadmin, only: [:destroy, :new, :create]
+
+  def index
+    @chairs = Chair.all
+  end
+
+  # Superadmin tasks:
+  def destroy
+    @chair.destroy
+    respond_to do |format|
+      format.html { redirect_to chairs_path, notice: 'Chair was successfully destroyed.' }
+      format.json { render json: @chair.errors, status: :unprocessable_entity }
+    end
+  end
+
+  def new
+    @chair = Chair.new
+  end
+
+  def create
+    @chair = Chair.new(chair_params)
+
+    success = false
+    admin = User.find_by(id: params[:admin_user])
+    representative = User.find_by(id: params[:representative_user])
+
+    if (admin && representative) && admin != representative
+      unless admin.is_wimi? || representative.is_wimi?
+        c1 = ChairWimi.new(:admin => true, :chair => @chair, :user => admin, :application => 'accepted')
```

Comment:
Können wir uns noch darauf verständen, symbol: statt :symbol => zu verwenden? Das spart schon zwei Chars pro Anwendung :) In den Code Styles wird das vorgeschlagen